doc/build-helpers/special.md

```diff
@@ -3,6 +3,7 @@
 This chapter describes several special build helpers.
 
 ```{=include=} sections
+special/fakenss.section.md
```

Comment:
I regret that someone at some point named a whole directory so subjectively.
The only thing that's special about Nix stuff is that the outputs are [utterly boring](https://nixos.org/manual/nix/stable/store/file-system-object).

Reply:
Added this as a note for myself for future work, because it would involve changing a bunch of links too (which is doable, just not something I want to do right now).